starknet-foundry/crates/cast/tests/e2e/mod.rs

```diff
@@ -3,4 +3,5 @@ mod declare;
 mod deploy;
 mod invoke;
 mod main_tests;
-mod multicall;
+mod multicall_new;
+mod multicall_run;
```

Comment:
Why seperate files?

Reply:
And why not? I think this decomposition is good, we can clearly distinguish tests, if I put all of these tests in one file, there would be a lot of them in one place, quite messy if u ask me.

Reply:
Multicall is a separate command, and we should treat it like that. All the other subcommands have one test file, and we should stick to this convention. Having 9 tests in one file is something I wouldn't call messy, especially if they are grouped :+1:

Reply:
Yes, all subcommands have their separate files, `multicall new` and `multicall run` are subcommands as well. I think `new` and `run` are different enough to have 2 files for them. Not sure what's the downside of having yet one file more either to be honest. Sorry, but I'd stay with separate files, what do you think @war-in?

Reply:
If you really insist on that, then we can consider finding some middle ground by changing the files structure to something like this:

```
├── declare.rs
├── deploy.rs
├── main_tests.rs
(...)
├── mod.rs
├── multicall
│   ├── new_tests.rs
│   └── run_tests.rs
```

What do you think?
I personally prefer a different approach with the file structure, as I believe it would be more beneficial in the long run. Let's discuss this further to ensure we find the best solution for the project.

Reply:
[folder structure](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/2f09acfafc401528b6a3d9d7328cc956cf5febb8)